src/components/Book.js

```diff
@@ -1,28 +1,61 @@
+/* eslint-disable react/self-closing-comp */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/booksSlice';
+import ProgressBar from './progressBar';
+import './Book.css';
+
+const categoriesBook = ['action', 'comdy', 'thriller', 'history', 'family', 'fantecy', 'joke'];
+const randomCategories = Math.floor(Math.random() * categoriesBook.length);
+const randomItem = categoriesBook[randomCategories];
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   return (
-    <>
-      <p key={book.id}>
-        <strong>{book.title}</strong>
-        {' '}
-        <span>by</span>
-        {' '}
-        <strong>{book.author}</strong>
-      </p>
-      <button
-        type="button"
-        onClick={() => {
-          dispatch(removeBook(book.id));
-        }}
-      >
-        Delete
-      </button>
-    </>
+    <div className="book-container">
+      <div className="book-intial--info" key={book.id}>
+        <div className="title-author">
+          <h4 className="category-random">{randomItem}</h4>
+          <h3 className="title">{book.title}</h3>
+          <h3 className="author">{book.author}</h3>
+        </div>
```

Comment:
- [ ] Great job so far @MMhaidari :muscle: Your project looks great :smile: . There are just a few minor adjustments you need to make :wink: 
- [ ] I have made sure to add short notes of the changes you need to make in the image below :+1: 
- Your Design 
![image](https://user-images.githubusercontent.com/102699958/232209852-6be629e9-452c-42b7-b5bc-31aa328e628a.png)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/232209986-0ea46e6f-4b47-438c-9842-c4069900fa40.png)

Reply:
@NahnahAJ dear regarding the update progress button if that shows the design like that which seems to be wrong is because you are using a small screen. and to remind you this project was not supposed to be responsive